README.md

```diff
@@ -202,12 +176,6 @@ You can deploy this project using:
 - GitHub: [@ijosue16](https://github.com/ijosue16)
 
 
-**👤 Author2**
-
-- GitHub: [@maccony202](https://github.com/maccony202)
-
-
-
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
- [ ] OPTIONAL
Kindly endeavor to fill the FAQ section with at least 2 Q&A(s), on _line 234_ of your ReadMe.